### DISCUSSION TITLE
egrep is obsolescent

### DISCUSSION THREAD

**Discussion by Nadhum** [ohmyzsh/ohmyzsh#11159] [Help] [answered]
I started getting warnings recently about these 
`egrep: warning: egrep is obsolescent; using grep -E`

https://github.com/ohmyzsh/ohmyzsh/blob/12669f29f0843b8b980dd137f150a74511f88842/lib/grep.zsh#L27-L28

Should these aliases be removed by default?

**Comment by carlosala** [comment, marked as answer]
We should remove those aliases probably, or at least change them. What do you think @mcornella?

**Reply by mcornella** [reply]
We can keep the aliases but use `grep -E` instead of the `egrep` command, and similarly for the `fgrep` alias.

**Reply by Nadhum** [reply]
Cool, will create an issue.
